force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -253,14 +323,21 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer sumPositiveIntegers() {
         List<Integer> numbers = new List<Integer>{-1, 2, -3, 4, -5, 6}; //DO NOT CHANGE
+        // Loop through the list of integers 
 
-        // Loop through the list of integers
+        Integer sum = 0;
+	for (Integer i = 0; i < numbers.size(); i++) {
+        Integer num = numbers[i];
             // if the number is negative skip this iteration
-        
-
-        return null; // Replace null with the variable you used to store the result
-    }
-
+            if (num <= 0) {
+                continue;
+            }
+            sum += num;
+            System.debug('Sum of Positive ' +sum);
+
+        //return null; // Replace null with the variable you used to store the result
+    }   
+}
```

Comment:
This wont compile. Missing a return statement. You don't need the continue condition here. 

` for (Integer num : numbers) {
            if(num > 0){
                sum += num;
            }

        }
        return sum;`